### PR TITLE
Add Com_Error and Com_PrintError to plugin api

### PR DESCRIPTION
**Pull request by T-Maxxx**

### CODE REVIEW

File: src_plug/alltests/main.cpp [hunk 1]
```diff
@@ -9,6 +9,8 @@ void OnPluginLoad(EPluginLoadingResult* pResult_)
 {
     trap_Com_Printf("[alltests]: Hello, %s world!\n", "formatted");
     *pResult_ = PLR_OK;
+	trap_Com_PrintError("[alltests]: Com_PrintError(): Test Com_PrintError %d\n", 1);
+	trap_Com_Error("[alltests]: Com_Error(): Test Com_Error %d\n", 2);
```
Comment (T-Maxxx) [inline src_plug/alltests/main.cpp]: Com_Error надо закомментировать. Я верю, что функция работает, но она убьёт весь процесс при вызове.

File: src_mod/phandler/dispatcher.cpp [hunk 2]
```diff
@@ -19,6 +19,12 @@ void SysCallDispatcher(const EAPICode Code_, ...)
         case AC_Com_Printf:
             Com_Printf(VARG(0, const char*));
             break;
+		case AC_Com_Error:
+			Com_Error(0,VARG(0, const char*));
```
Comment (T-Maxxx) [inline src_mod/phandler/dispatcher.cpp]: Стиль - пробел после запятой.

File: src_mod/plugin/traps.h [hunk 3]
```diff
@@ -18,4 +18,28 @@ inline void trap_Com_Printf(const char* const Format_, ...)
     va_end(ap);
 
     syscall(AC_Com_Printf, msg);
+}
+
+inline void trap_Com_Error(const char* const Error, ... ) {
```
Comment (T-Maxxx) [inline src_mod/plugin/traps.h]: Стиль - скобка с новой строки.

File: src_mod/plugin/traps.h [hunk 4]
```diff
@@ -18,4 +18,28 @@ inline void trap_Com_Printf(const char* const Format_, ...)
...
+    va_list ap;
+    va_start(ap, Error);
+
+    vsnprintf(msg, sizeof(msg), Error, ap);
+    va_end(ap);
+
+	syscall(AC_Com_Error, msg);
+}
+
+inline void trap_Com_PrintError(const char* const Fmt, ... ) {
```
Comment (T-Maxxx) [inline src_mod/plugin/traps.h]: Стиль - скобка с новой строки.

**Comment by D4edalus** [comment]
@T-Maxxx could you please comment english in future, its tedious to translate. thanks

**Comment by T-Maxxx** [comment]
@D4edalus That's small changes to my branch from other russian developer. He wants to help us with development.

**Comment by pdbx** [comment]
@T-Maxxx Извини за табуляции, давно не использовал github

**Comment by D4edalus** [comment]
i certainly dont wanna force you to anything, but i think its a simple form of politeness. neither of our native languages is english - yet we speak english so everyone is able to undestand what we are saying. same on the forums.

**Comment by pdbx** [comment]
@D4edalus Sorry dude, now I will speak English

**Pull request merged**